src/js/themes/base.d.ts

```diff
@@ -840,6 +840,9 @@ export interface ThemeType {
     };
   };
   dateInput?: {
+    button?: {
+      margin?: MarginType;
```

Comment:
Technically we're only supporting "left" and "right" at the moment right?

We won't cause any errors with this, so it's okay but if we wanted to be more specific we should probably break it down

Reply:
Changed the type to just support passing a left and right margin